test/device-test/src/test_smarteeprom.c

```diff
@@ -87,5 +88,8 @@ int main(void)
         _screen_print_debug("SmartEEPROM enabled.\nRebooting.", 2000);
         _reset_mcu();
     }
+    for (;;) {
+        screen_process();
+    }
```

Comment:
without this, there is a `undefined reference to screen_process` linking error. I have no idea why this would happen here.

I just know:
- somehow it is LTO-optmized away
- Rust code calls it, which is why there is a reference error

Unclear to me is:
- Why it is optimized away. This does not happen for other C functions that are solely used in Rust
- Why it does not happen for other device tests, like `test_usb_composed_ep_in.c`, which also does not use any workflow/screen_process stuff.

Would be good to know, but not a high prio, since it is only a device test.

Reply:
Please add a comment in the code? So, we'll remember to come back to this later maybe. A comment wouldn't hurt anyway.